tests/unit/push_pair_00.tzt

```diff
@@ -0,0 +1,3 @@
+code { PUSH (pair string string string) (Pair "Hello" "World" "!") ; DROP } ;
+input  { } ;
+output { }
```

Comment:
Consider adding testcases for the mixed use of syntax (for both data and type), i.e. `(pair string string string)` with `(pair string (pair string string))`, etc.

Reply:
Done.